cmd/oras/internal/display/status/tty_console_test.go

```diff
@@ -78,3 +83,91 @@ func Test_TTYPullHandler_TrackTarget(t *testing.T) {
 		}
 	})
 }
+
+func TestTTYCopyHandler_OnMounted(t *testing.T) {
+	pty, slave, err := testutils.NewPty()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer slave.Close()
+	ch := NewTTYCopyHandler(slave)
+	_, err = ch.StartTracking(&testGraphTarget{memory.New()})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer ch.StopTracking()
+
+	if err = ch.OnMounted(ctx, mockFetcher.OciImage); err != nil {
+		t.Errorf("OnMounted() should not return an error: %v", err)
+	}
+
+	if err = testutils.MatchPty(pty, slave, "\x1b[?25l\x1b7\x1b[0m"); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestTTYCopyHandler_OnCopySkipped(t *testing.T) {
+	pty, slave, err := testutils.NewPty()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer slave.Close()
+	ch := NewTTYCopyHandler(slave)
+	_, err = ch.StartTracking(&testGraphTarget{memory.New()})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer ch.StopTracking()
+
+	if err = ch.OnCopySkipped(ctx, mockFetcher.OciImage); err != nil {
+		t.Errorf("OnCopySkipped() should not return an error: %v", err)
+	}
+
+	if err = testutils.MatchPty(pty, slave, "\x1b[?25l\x1b7\x1b[0m"); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Should synchronizely stop tracking and match output
```suggestion

	if err = ch.OnCopySkipped(ctx, mockFetcher.OciImage); err != nil {
		t.Errorf("OnCopySkipped() should not return an error: %v", err)
	}

	ch.StopTracking()
	if err = testutils.MatchPty(pty, slave, "Skipped", mockFetcher.OciImage.MediaType, mockFetcher.OciImage.Size, "100%"); err != nil {
		t.Fatal(err)
	}
```

Reply:
Can be applied to other tests in this file.